commcare_translations.py

```diff
@@ -36,7 +37,17 @@ def load_translations(lang, version=1, commcare_version=None):
         return {}
 
     paths_to_try = []
-    if commcare_version:
+    if commcare_version == 'latest':
+        files = listdir(normpath(join(__file__, "../historical-translations-by-version/")))
```

Comment:
`normpath` deals with it - `rel_path` is `../historical-translations-by-version/2.35-messages_en-2.txt` and `normpath(join(__file__, rel_path))` turns it into `submodules/commcare-translations/historical-translations-by-version/2.35-messages_en-2.txt`